definitions/materialized/neftyquest_leaderboard_template.sql

```diff
@@ -0,0 +1,75 @@
+CREATE MATERIALIZED VIEW IF NOT EXISTS nefy_quest_leaderboard_{{quest_id}} AS
+SELECT rank() OVER (ORDER BY experience DESC) "rank",
+       account,
+       total_sold,
+       total_bought,
+       items_sold,
+       items_bought,
+       (total_collected / {{total_to_collect}}::numeric) completion_percentage,
+       experience,
+       tokens.token_symbol symbol,
+       tokens.token_precision symbol_precision
+FROM (
+     SELECT
+       COALESCE(seller, buyer) account,
+        COALESCE(nefty_sells.total_sold, 0) total_sold,
+        COALESCE(nefty_buys.total_bought, 0) total_bought,
+        COALESCE(nefty_sells.sold_items, 0) items_sold,
+        COALESCE(nefty_buys.bought_items, 0) items_bought,
+        COALESCE(templates_owned.total, 0) total_collected,
+        COALESCE(nefty_sells.symbol, nefty_buys.symbol) AS symbol,
+        (
+                (CASE COALESCE(templates_owned.total, 0) WHEN {{total_to_collect}} THEN {{completion_multiplier}} ELSE 1 END) * 100 *
+                (
+                        (COALESCE(nefty_sells.total_sold, 0) + COALESCE(nefty_buys.total_bought, 0)) / {{volume_threshold}} +
+                        (COALESCE(nefty_sells.sold_items, 0) + COALESCE(nefty_buys.bought_items, 0)) * {{points_per_asset}}
+                )
+        ) experience
```

Comment:
maybe we should make a function so it's more readable, like the one used to filter by attributes in blends

Reply:
Mmmm.... I would keep it in a regular query if it's only for readability